docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -252,6 +252,8 @@ this table:
     - ``VARBINARY``
   * - ``DATE``
     - ``DATE``
+  * - ``TIMESTAMP_NTZ``
```

Comment:
Delta spec has this without underscore.

```suggestion
  * - ``TIMESTAMPNTZ`` (``TIMESTAMP_NTZ``)
```

??

Reply:
`TIMESTAMP_NTZ ` is the type name in Spark. `TIMESTAMPNTZ` is just a feature name in Delta.

Reply:
it might be that Delta type system / type names != Spark type names